docs/src/test.md

```diff
@@ -45,3 +45,72 @@ pytest -svv forge/test/mlir/test_ops.py::test_add
 ```
 
 > - The `-svv` flag is optional and used to display more information about the test run.
+
+## Single operator E2E tests
+
+Single operator E2E tests consists of pre configured collections of in-depth tests for each operator according to test plan.
+Tests include small models consisting of single operator with or without combination with few other operators.
+More details about test plan available on [Test template page](https://github.com/tenstorrent/tt-forge-fe/blob/main/forge/test/operators/test_plan_template.md)
+
+To run all tests from test plan use `test_plan` method:
+
+```sh
+pytest -svv forge/test/operators/pytorch/test_all.py::test_plan
+```
+
+Full list of supported query criteria
+
+| Parameter          | Description                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------ |
+| OPERATORS          | List of operators                                                                          |
```

Comment:
Do we plan to populate these now, or this planed a bit latter?

Reply:
Live documentation of all supported operators is now available via `print_query_docs` function from `test_commands.sh` helper scripts.

The documentation we might remove from here as redundant.

Reply:
Documentation updated to include `print_query_docs`